
xlocale.h is deprecated and not available in glibc-2.26

on musl there is locale.h, too


--- src/framework/mlt_property.h.orig	2017-08-30 13:55:55.676748750 +0200
+++ src/framework/mlt_property.h	2017-08-30 13:59:27.660276105 +0200
@@ -31,7 +31,11 @@
 #endif
 
 #if defined(__GLIBC__) || defined(__APPLE__) || (__FreeBSD_version >= 900506)
-#include <xlocale.h>
+#  if __GLIBC_MINOR__ >= 26 && !defined(__APPLE__)
+#    include <locale.h>
+#  else  
+#    include <xlocale.h>
+#  endif
 #else
-typedef char* locale_t;
+#include <locale.h>
 #endif

